src/main/java/com/github/manolo8/darkbot/core/manager/HeroManager.java

```diff
@@ -195,6 +196,9 @@ public boolean attackMode() {
     public boolean attackMode(Npc target) {
         if (target == null) return attackMode();
         Config.ShipConfig config = this.main.config.GENERAL.OFFENSIVE;
+        if (target.npcInfo.extra.has(NpcExtra.OPPOSITE_CONFIG)) {
+            config.CONFIG = 1 - config.CONFIG; // Toggle between 0 and 1
```

Comment:
Make a ternary that if the config is one, the value is set to 0.

`config.CONFIG = config.getConfiguration() == Configuration.FIRST ? Configuration.SECOND.ordinal() : Configuration.FIRST.ordinal();`